consensus/src/block_graph.rs

```diff
@@ -4809,6 +4809,7 @@ mod tests {
             endorsement_count: 8,
             block_db_prune_interval: 1000.into(),
             max_item_return_count: 1000,
+            max_gas_per_block: 10,
```

Comment:
It seems like a tiny default value, @adrien-zinger any idea on what would be a good one ?